backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2746,6 +2746,117 @@ databaseChangeLog:
+                p.lookup_id,
+                get_census_dob_group(p.birth_date) AS census_dob_group,
+                p.preferred_language,
+                p.test_result_delivery_preference
+              FROM ${database.defaultSchemaName}.person p
+        - sql: |
+            GRANT SELECT ON ${database.defaultSchemaName}.person_no_phi_view TO ${noPhiUsername};
+      rollback:
+        - dropView:
+            viewName: person_no_phi_view
+        - createView:
+            viewName: person_no_phi_view
+            remarks: A subset of the person table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: |
+              SELECT
+                p.internal_id,
+                p.created_at,
+                p.created_by,
+                p.updated_at,
+                p.updated_by,
+                p.is_deleted,
+                p.organization_id,
+                p.facility_id,
+                p.race,
+                p.gender,
+                p.ethnicity,
+                p.county,
+                p.state,
+                p.employed_in_healthcare,
+                p.role,
+                p.resident_congregate_setting,
+                p.tribal_affiliation,
+                p.lookup_id,
+                get_census_dob_group(p.birth_date) AS census_dob_group,
+                pp.preferred_language,
+                pp.test_result_delivery
+              FROM ${database.defaultSchemaName}.person p
+              JOIN patient_preferences pp on pp.internal_id=p.internal_id
+        - sql: |
+            GRANT SELECT ON ${database.defaultSchemaName}.person_no_phi_view TO ${noPhiUsername};
+  - changeSet:
+      id: denormalize-patient_preferences-3
+      author: adam@skylight.digital
+      changes:
+        - dropTable:
+            tableName: patient_preferences
+      rollback:
+        - createTable:
+            tableName: patient_preferences
+            columns:
+              - column:
+                  name: internal_id
+                  type: uuid
+                  remarks: The internal database identifier of the Person to whom these preferences belong
+                  constraints:
+                    foreignKeyName: fk__patient_preferences__person
+                    references: person
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: preferred_language
+                  type: text
+                  remarks: Tracks the individual's preferred language
+              - column:
+                  name: test_result_delivery
+                  type: ${database.defaultSchemaName}.TEST_RESULT_DELIVERY
+                  remarks: Tracks the individual's preference for receiving test results
+              - column: *created_at_column
+              - column: *created_by_column
+              - column: *updated_at_column
+              - column: *updated_by_column
```

Comment:
This diff looks a little strange, but what's going on here is that the rollback includes copying the data back from `person` into `patient_preferences`, a step that was _also_ included as the rollback in the previous migration